src/components/PlanetInfo/planetInfo.css

```diff
@@ -4,11 +4,22 @@
   justify-content: center;
 }
 
+.planet-info-title {
+  font-size: 40px;
+}
+
 .planet-info-img {
   width: 200px;
   align-self: center;
 }
 
+.planet-info-box {
+  border: 5px solid #D0D7DF;
+  width: 70%;
+  align-self: center;
```

Comment:
I think this `align-self: center` line might not be necessary given the setup of the flexbox it's in. (And similarly for the `align-self` line in the section above.)

Reply:
Okay, my bad, looks like this is needed after all!